odd-platform-ui/src/components/shared/TestReportTypeItem/TestReportTypeItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import cx from 'classnames';
+import {
+  styles,
+  StylesType,
+} from 'components/shared/TestReportTypeItem/TestReportTypeItemStyles';
+
+export type DataSetTestReportTypeNames =
```

Comment:
Very similar to DataQualityTestRunStatusEnum that is being generated from api spec.